.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    container:
+      image: pgbrodrick/isofit
```

Comment:
Should this be switched to `jammont/isofit` in preparation for the more official `isofit/isofit` image that'll be posted soon? We can wait to include that in the next PR, but something to consider.

Reply:
Absolutely, I agree! It's just a legacy from previous times here, since I started to work on this workflow some months ago. I'd suggest that we wait until #328 is merged and I then update the image in this PR before we merge it in. I don't think we need a new PR for this.